docs/manual/kinds/keysight_ixia-c-one.md

```diff
@@ -0,0 +1,121 @@
+    curl -kL https://clab-ixia-c-ixia-c-one/config
+
+    # fetch flow metrics
+    curl -kL https://clab-ixia-c-ixia-c-one/results/metrics -d '{"choice": "flow"}'
+    ```
+
+## SDK
+Client SDK for configuring ixia-c is available in various languages, most prevalent being [gosnappi][gosnappi] and [snappi][snappi].
+
+The example below demonstrates how [gosnappi][gosnappi] - Go-based SDK client - can be used to configure/run a traffic test along with evaluating results.
+
+IPv4 Forwarding test is setup in the following way:
+
+Endpoints: `OTG 1.1.1.1 -----> 1.1.1.2 DUT 2.2.2.1 ------> OTG 2.2.2.2`  
+Static Route on DUT: `20.20.20.0/24 -> 2.2.2.2`  
+TCP flow from OTG: `10.10.10.1 -> 20.20.20.1+`
+
+1. [install](https://go.dev/doc/install) go and init a module
+```bash
+mkdir tests && cd tests
+go mod init tests
+```
+1. Download gosnappi release. gosnappi version needs to be compatible to a given release of ixia-c and
+can be checked at https://github.com/open-traffic-generator/ixia-c/releases
+```bash
+go get github.com/open-traffic-generator/snappi/gosnappi@v0.7.18
+```
+1. download a [basic IPv4 forwarding test](https://github.com/open-traffic-generator/snappi-tests/blob/b2b0d32e8d19589dc69ebd71eb5929d5f3c908f2/scripts/ipv4_forwarding.go)
+```
+curl -LO https://raw.githubusercontent.com/open-traffic-generator/snappi-tests/main/scripts/ipv4_forwarding.go
+```
+1. run the test with MAC address of the `1.1.1.2` interface which is located on the DUT as per the test diagram.
+```
+go run ipv4_forwarding.go -dstMac="<MAC address>"
+```
+
+## Interfaces mapping
+ixia-c-one container uses the following mapping for its linux interfaces:
+
+* `eth0` - management interface connected to the containerlab management network
+* The other interfaces (eth1, eth2, etc) are the data interfaces acting as traffic generation ports
+
+When containerlab launches ixia-c-one node, it will set IPv4/6 addresses as assigned by docker to the `eth0` interface and ixia-c-one node will boot with that addresses configured.
+
+Data interfaces `eth1+` need to be configured with IP addressing manually if needed (as in the Layer3 forwarding test example).
+IP addresses are required when the test port needs to reply to ARP/ND queries from the Device Under Test. These addresses can be configured on non-`eth0` ports.
+
+Examples below show how test designer can configure IP address on eth1 data port of a parent container named `ixiac-one`:
+
+=== "IPv4 configuration"
+    ```bash
+    # set ipv4 addr
+    docker exec -it ixiac-one bash -c "./ifcfg add eth2 2.2.2.2 24"
+
+    # remove ipv4 addr
+    docker exec -it ixiac-one bash -c "./ifcfg del eth2 2.2.2.2 24"
+    ```
+=== "IPv6 configuration"
+    ```bash
+    # set ipv6 addr on eth1 iface
+    docker exec -it ixiac-one bash -c "./ifcfg add eth1 11::1 64"
+
+    # remove ipv6 addr on eth1 iface
+    docker exec -it ixiac-one bash -c "./ifcfg del eth1 11::1 64"
+    ```
```

Comment:
Hi @rudranil-das @ashutshkumr 
this has to be some typo here when you refer to `./ifcfg` script which assumed to be available in a parent ixia-c container. 
It seems like it doesn't?

It would actually be good to have a script `dockerifcfg` inside the parent dind container that will take in the same arguments and call `docker exec ... ifcfg add eth1 ...` behind the hood.

Right now I think I need to fix it and call another `docker exec` inside the main `docker exec` context

Reply:
@hellt `./ifcfg` is a script that is available in parent ixia-c-one container, and it already takes care of propagating commands to child container.

Reply:
what is the path for this patched ifcfg script? 
Because once I enter in the parent container image I have the global ifcfg script only

```
bash-5.1# cat /sbin/ifcfg | grep docker
```

or maybe was added recently?

Reply:
The path is `/home/keysight/ixia-c-one/ifcfg`. It's available for the release provided in clab yaml file.

Reply:
gotcha. I was using 2610 version which didn't have this script. All is good when udpated ixia image